ui-admin/src/portal/MailingListView.test.tsx

```diff
@@ -73,3 +73,20 @@ test('delete button shows confirmation', async () => {
   await userEvent.click(screen.getByText('Remove'))
   expect(screen.getByText('This operation CANNOT BE UNDONE.')).toBeInTheDocument()
 })
+
+test('sorts by join date by default', async () => {
+  jest.spyOn(Api, 'fetchMailingList').mockImplementation(() => Promise.resolve(contacts))
+  const portalContext = mockPortalContext()
+  const portalEnv = portalContext.portal.portalEnvironments[0]
+  const { RoutedComponent } =
+      setupRouterTest(<MailingListView portalContext={portalContext} portalEnv={portalEnv}/>)
+  render(RoutedComponent)
+  await waitFor(() => {
+    expect(screen.getByText('person1')).toBeInTheDocument()
+  })
+
+  const joined = screen.getByText('Joined')
+
+  // "Joined" text is two parents away from the <th> element
+  expect(joined.parentElement?.parentElement?.getAttribute('aria-sort')).toBe('descending')
```

Comment:
using `closest` will make it so you don't have to hardcode the parent distance
```suggestion
 expect(joined.closest('[aria-sort]')?.getAttribute('aria-sort')).toBe('descending')
```

Reply:
Ah smart. I felt like there should have been a better way. Thanks!